src/App.js

```diff
@@ -1,17 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLightbulb } from '@fortawesome/fontawesome-free-solid';
 
+import styled, { ThemeProvider } from 'styled-components';
+import { lightTheme, darkTheme } from './components/Themes';
 import Home from './pages/home';
+import Button from './components/Button';
+
+const App = () => {
+  const [theme, setTheme] = useState('light');
+  const themeToggler = () => {
+    // eslint-disable-next-line no-unused-expressions
+    theme === 'light' ? setTheme('dark') : setTheme('light');
+  };
```

Comment:
Ideally theme should be move to context or redux (preferably context). It's fine for, But do keep in mind.